sql-cli/sql_cli/__main__.py

```diff
@@ -294,6 +294,31 @@ def init(
     rprint("Initialized an Astro SQL project at", project.directory)
 
 
+@app.command(
+    cls=AstroCommand,
+    help="""Deploy workflows to Astro Cloud.""",
+)
+def deploy(
+    workflow_name: str = typer.Option(  # skipcq: PYL-W0613
+        default=None,
+        show_default=False,
+        help="name of the workflow directory within workflows directory.",
+    ),
+    env: str = typer.Option(  # skipcq: PYL-W0613
+        metavar="environment",
+        default="default",
+        help="environment to deploy",
+    ),
+    project_dir: Path = typer.Option(  # skipcq: PYL-W0613
+        None, dir_okay=True, metavar="PATH", help="(Optional) Default: current directory.", show_default=False
+    ),
+) -> None:
+    rprint(
+        "Please use the astro-cli to deploy. See https://docs.astronomer.io/astro/cli/overview for details."
```

Comment:
I think it is worth it for us to point to the SQL CLI docs, which will have deploy for SQL workflows:
astronomer.io/docs/astro/cli/sql-cli.html

The

Reply:
The? 🤔

Reply:
You probably mean this link https://docs.astronomer.io/astro/cli/sql-cli or?

Reply:
```suggestion
        "Please use the Astro CLI to deploy. See https://docs.astronomer.io/astro/cli/sql-cli for details."
```